### DISCUSSION TITLE
Add attachment deduplication tool

### DISCUSSION THREAD

**Discussion by benanhalt** [specify/specify7#5514] [Feature Requests]
Need a tool for finding duplicate attachments and merging them (either automatic detection of duplicates or a manual feature of selecting two attachments and clicking "merge")

**Comment by maxpatiiuk** [comment]
Let attachment picket idea be tracked by #2999, and reword this to be for finding duplicate attachments only

**Comment by grantfitzsimmons** [comment]
It would be very nice to have a tool that allowed you to link an existing attachment from another record with a new record.

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/open-discussion-on-associated-media-management-in-specify-7/2156/3

**Comment by grantfitzsimmons** [comment]
Need to make sure users can select an existing image
